samples/cpp/benchmark_app/utils.cpp

```diff
@@ -895,3 +897,17 @@ std::string parameter_name_to_tensor_name(const std::string& name,
     throw std::runtime_error("Provided I/O name \"" + name +
                              "\" is not found neither in tensor names nor in nodes names.");
 }
+
+void processBinaryFile(std::ifstream& binaryFile, const std::string& fileName, const unsigned long inputSize) {
```

Comment:
Right, refactored the name. The verify method was extracted from the inputs_filling.cpp when I was adding the npy loading functionality, so I assume it is finished.  It looks more or less ok - there isn't much to verify in case of pure binary files except the byte size